pallets/gear/src/lib.rs

```diff
@@ -629,7 +629,7 @@ pub mod pallet {
                 code,
                 schedule.instruction_weights.version,
                 Some(module),
-                false,
+                true,
```

Comment:
there is also 2 usage of `Code::new_raw` and shouldn't they also be with true flag?

Reply:
@ukint-vs please take a closer look to this

Reply:
Discussed with @ukint-vs, we will deal with all of related to `_raw` calls after benchmark reworking

Reply:
we can merge it as it is and refactor benchmarks later with @grishasobol

Reply:
ok